wso2-micro-integrator-mixin/dashboards/API_Metrics.json

```diff
@@ -0,0 +1,810 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": "-- Grafana --",
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "type": "dashboard"
+      }
+    ]
+  },
+  "description": "This Dashboard provides an overview of API services deployed in Enterprise Integrator Cluster.",
```

Comment:
Appreciate the generous use of descriptions!

Reply:
This is helpful for sure!